packages/create-ponder/src/index.ts

```diff
@@ -207,30 +207,40 @@ export async function run({
 
   const targetPath = path.join(process.cwd(), projectPath);
 
+  let link: string | undefined = undefined;
   if (templateMeta.id === "etherscan") {
-    let link = options.etherscanContractLink;
+    link = options.etherscanContractLink;
     if (!link) {
       const result = await prompts({
         type: "text",
         name: "link",
-        message: "Enter an Etherscan contract link",
+        message: "Enter a block explorer contract link",
```

Comment:
Do you also want to rename the cli option?

Reply:
Not yet